libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/binder/map/FreeDriveMapBinder.kt

```diff
@@ -22,6 +23,11 @@ internal class FreeDriveMapBinder(
                 value,
                 navigationViewContext.viewModel.locationViewModel,
             ),
+            RouteLineComponent(
+                value,
+                navigationViewContext.routeLineOptions,
+                navigationViewContext.viewModel.routesViewModel
+            ),
```

Comment:
Was this added just for test purposes? `RouteLineComponent` should not be a part of `FreeDrive` or is this change intentional?

Reply:
Okay I see why this is important and required.